CHANGELOG.md

```
@@ -1,45 +1,29 @@
-
+- **#178:** dependabot workflow ([#179](https://github.com/VilnaCRM-Org/website/issues/179)) ([bc20712](https://github.com/VilnaCRM-Org/website/commit/bc207126629fda72074a7b5d49d80762bfa0b3b6)), closes [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178) [#178](https://github.com/VilnaCRM-Org/website/issues/178)
 
 # [1.4.0](https://github.com/VilnaCRM-Org/website/compare/v1.3.0...v1.4.0) (2025-04-28)
 
-
 ### Features
 
-* **#28:** redesign "For Whom" block  ([#171](https://github.com/VilnaCRM-Org/website/issues/171)) ([cd224fe](https://github.com/VilnaCRM-Org/website/commit/cd224fed3a45a086dc8db1ec89cb6be4d62640fb)), closes [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28)
-
-
+- **#28:** redesign "For Whom" block ([#171](https://github.com/VilnaCRM-Org/website/issues/171)) ([cd224fe](https://github.com/VilnaCRM-Org/website/commit/cd224fed3a45a086dc8db1ec89cb6be4d62640fb)), closes [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28) [#28](https://github.com/VilnaCRM-Org/website/issues/28)
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 100; Actual: 833

(MD013, line-length)